Pod/Classes/MPOFaceServiceClient.h

```diff
@@ -239,7 +313,16 @@ typedef void (^MPOFaceListMetadataArrayCompletionBlock)(NSArray<MPOFaceListMetad
  *  @param completion    completion handler
  *  @return the persons, NSArray containing MPOPerson objects
  */
-- (NSURLSessionDataTask *)getPersonsWithPersonGroupId:(NSString *)personGroupId completionBlock:(MPOPersonArrayCompletionBlock)completion;
+- (NSURLSessionDataTask *)getPersonsWithPersonGroupId:(NSString *)personGroupId completionBlock:(MPOPersonArrayCompletionBlock)completion NS_DEPRECATED_IOS(7.0, 7.0);
```

Comment:
This function get all the persons in the specified person group, will not be de deprecated. Please remove "completion NS_DEPRECATED_IOS(7.0, 7.0)"